.github/workflows/go.yml

```diff
@@ -23,14 +23,16 @@ jobs:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
 
-    - name: Build
+    - name: Build and upload coverage to Codecov
       env:
         CGO_ENABLED: 0
         GO111MODULE: on
+        CODECOV_TOKEN: "d8ab4164-f215-4ccb-a14a-bacf23e27258"
```

Comment:
Have to create an account in codecov.io for direct-csi and upload the token here

Reply:
CODECOV_TOKEN="9128c679-541f-4cf7-b9a7-6b7d9c95f774"